src/AspNet.Security.OAuth.BattleNet/BattleNetAuthenticationDefaults.cs

```diff
@@ -0,0 +1,90 @@
+﻿/*
+ * Licensed under the Apache License, Version 2.0 (http://www.apache.org/licenses/LICENSE-2.0)
+ * See https://github.com/aspnet-contrib/AspNet.Security.OAuth.Providers
+ * for more information concerning the license and the contributors participating to this project.
+ */
+
+using Microsoft.AspNet.Authentication;
+using Microsoft.AspNet.Authentication.OAuth;
+
+namespace AspNet.Security.OAuth.BattleNet {
+    /// <summary>
+    /// Default values used by the Battle.net authentication middleware.
+    /// </summary>
+    public static class BattleNetAuthenticationDefaults {
+        /// <summary>
+        /// Default value for <see cref="AuthenticationOptions.AuthenticationScheme"/>.
+        /// </summary>
+        public const string AuthenticationScheme = "BattleNet";
+
+        /// <summary>
+        /// Default value for <see cref="OAuthAuthenticationOptions.Caption"/>.
+        /// </summary>
+        public const string Caption = "BattleNet";
+
+        /// <summary>
+        /// Default value for <see cref="OAuthAuthenticationOptions.ClaimsIssuer"/>.
+        /// </summary>
+        public const string Issuer = "BattleNet";
+
+        /// <summary>
+        /// Default value for <see cref="OAuthAuthenticationOptions.CallbackPath"/>.
+        /// </summary>
+        public const string CallbackPath = "/signin-battlenet";
+
+        /// <summary>
+        /// Default value for <see cref="OAuthAuthenticationOptions.AuthorizationEndpoint"/> for US servers.
+        /// </summary>
+        public const string AuthorizationEndpointUS = "https://us.battle.net/oauth/authorize";
```

Comment:
Consider creating nested classes to avoid the region suffix: `BattleNetAuthenticationDefaults.US.AuthorizationEndpoint`